### PR TITLE
[jsk_fetch_robot] update README for develop/fetch branch

### PR DESCRIPTION
**Pull request by MiyabiTane**
I tried to setup the environment (For Remote PC) following the README, but I could not connect to the real robot (kinematic simulator launched).
I solved this problem by checking out the branch from `master` to `develop/fetch`.

(@mqcmd196  Thank you for helping me!)

### CODE REVIEW

File: jsk_fetch_robot/README.md [hunk 1]
```diff
@@ -50,8 +50,42 @@ wget https://raw.githubusercontent.com/jsk-ros-pkg/jsk_roseus/master/setup_upstr
...
+git fetch origin
+git checkout -b develop/fetch origin/develop/fetch
+```
+
+Next, you need to install `roseus_resume` package. For more information, please check [Affonso-Gui/roseus_resume](https://github.com/Affonso-Gui/roseus_resume).
+```
+cd ../../  # catkin_ws/src
+# clone euslisp
+wget https://raw.githubusercontent.com/jsk-ros-pkg/jsk_roseus/master/setup_upstream.sh -O /tmp/setup_upstream.sh
+bash /tmp/setup_upstream.sh -w ..
```
Comment (mqcmd196) [inline jsk_fetch_robot/README.md:63]:
```
wget https://raw.githubusercontent.com/jsk-ros-pkg/jsk_roseus/master/setup_upstream.sh -O /tmp/setup_upstream.sh
bash /tmp/setup_upstream.sh -w ..
```
have already written above.

File: jsk_fetch_robot/README.md [hunk 2]
```diff
@@ -50,8 +50,42 @@ wget https://raw.githubusercontent.com/jsk-ros-pkg/jsk_roseus/master/setup_upstr
...
+cd ../../jsk-ros-pkg/jsk_roseus/
+git remote add Affonso-Gui https://github.com/Affonso-Gui/jsk_roseus.git
+git fetch Affonso-Gui
+git checkout eus10
+cd ../../  # catkin_ws/src
+
+# clone roseus_resume
+git clone https://github.com/Affonso-Gui/roseus_resume.git
+```
+
```
Comment (mqcmd196) [inline jsk_fetch_robot/README.md:84]: I think these things can be merged into `.rosinstall`. Could you update `jsk_fetch_robot/jsk_fetch_user.rosinstall` instead?

**Review by mqcmd196 — CHANGES_REQUESTED** [review]

**Comment by tkmtnt7000** [comment]
Thank you for contribution!! @MiyabiTane 
This information should be also useful for semi-students.

File: jsk_fetch_robot/README.md (hunk 2 above)
Comment (mqcmd196) [reply jsk_fetch_robot/README.md:84]: The above means 
```
# clone roseus
wstool set jsk-ros-pkg/jsk_roseus --git https://github.com/jsk-ros-pkg/jsk_roseus.git -v master -u -y
cd euslisp/Euslisp/
# checkout euslisp/eus-handler
git remote add Affonso-Gui https://github.com/Affonso-Gui/EusLisp.git
git fetch Affonso-Gui
git checkout eus10
# checkout roseus/eus-handler
cd ../../jsk-ros-pkg/jsk_roseus/
git remote add Affonso-Gui https://github.com/Affonso-Gui/jsk_roseus.git
git fetch Affonso-Gui
git checkout eus10
cd ../../  # catkin_ws/src
# clone roseus_resume
git clone https://github.com/Affonso-Gui/roseus_resume.git
```

**Review by Affonso-Gui — CHANGES_REQUESTED** [review]
The simplest way to build `develop/fetch` with eus10 is probably the following.
```bash
mkdir -p catkin_ws/src
cd  catkin_ws/src
wstool init .                       
wstool set --git jsk-ros-pkg/jsk_robot https://github.com/jsk-ros-pkg/jsk_robot.git -v develop/fetch -y
wstool merge -t . https://raw.githubusercontent.com/jsk-ros-pkg/jsk_robot/master/jsk_fetch_robot/jsk_fetch_user.rosinstall.$ROS_DISTRO
wstool merge -t . https://gist.githubusercontent.com/Affonso-Gui/25518fef9dc7af0051147bdd2a94b116/raw/e3fcbf4027c876329801a25e32f4a4746200ddae/guiga_system.rosinstall
wstool update -t .
# To use eus10, furuschev script is required.
wget https://raw.githubusercontent.com/jsk-ros-pkg/jsk_roseus/master/setup_upstream.sh -O /tmp/setup_upstream.sh
bash /tmp/setup_upstream.sh -w ../ -p jsk-ros-pkg/geneus -p euslisp/jskeus
source /opt/ros/$ROS_DISTRO/setup.bash
rosdep install -y -r --from-paths . --ignore-src
cd ../
catkin build fetcheus jsk_fetch_startup
```
Before https://github.com/knorth55/jsk_pr2eus/pull/2 you also need to explicitly build `roseus_resume`.

I'll update the `roseus_resume` readme as well, that is not being very helpful.

**Comment by MiyabiTane** [comment]
@mqcmd196  @Affonso-Gui 
Thank you for your reviewing!!

Now, I think I do not need to change `jsk_fetch_robot/jsk_fetch_user.rosinstall` because `https://gist.githubusercontent.com/Affonso-Gui/25518fef9dc7af0051147bdd2a94b116/raw/e3fcbf4027c876329801a25e32f4a4746200ddae/guiga_system.rosinstall` includes `roseus_resume` package.
Is this understanding correct?

**Comment by Affonso-Gui** [comment]
@MiyabiTane Yes, I think both ways are acceptable.

I do prefer the `Affonso-Gui/gist` merge since it includes other eus10 related packages which would be hard to justify including in the fetcheus rosinstall, but will likely be needed by users running eus10 from source.

**Review by Affonso-Gui — APPROVED** [review]

**Review by mqcmd196 — APPROVED** [review]
LGTM

**Pull request merged**